<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Clase_Algoritmos</class>
 <widget class="QMainWindow" name="Clase_Algoritmos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>445</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>885</width>
    <height>445</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>885</width>
    <height>445</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(25, 34, 41);
</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::VerticalTabs</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(33, 46, 54);
border-top-color: rgb(42, 59, 71);
border-right-color: rgb(42, 59, 71);
border-buttom-color: rgb(42, 59, 71);
border-left-color: rgb(42, 59, 71);
color: rgb(16, 131, 214);
</string>
    </property>
    <property name="text">
     <string>Clase sobre Algoritmos de Ordenamiento</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-top-color: rgb(42, 59, 71);
border-right-color: rgb(42, 59, 71);
border-buttom-color: rgb(42, 59, 71);
border-left-color: rgb(42, 59, 71);
color: rgb(200, 205, 192);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="iD">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="imagen">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>391</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="texto">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>461</width>
      <height>291</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-top-color: rgb(42, 82, 158);
border-right-color:rgb(42, 82, 158);
border-buttom-color: rgb(42, 59, 71);
border-left-color: rgb(42, 82, 158);
color: rgb(200, 205, 192);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="questionLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(33, 46, 54);
background-color: rgb(36, 49, 59);
border-top-color: rgb(42, 59, 71);
border-right-color: rgb(42, 59, 71);
border-buttom-color: rgb(42, 59, 71);
border-left-color: rgb(42, 59, 71);
color: rgb(16, 131, 214);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>370</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="1">
      <widget class="QPushButton" name="irQuiz1">
       <property name="styleSheet">
        <string notr="true">border-style: solid;
border-width: 1px;
;
background-color: rgb(0, 120, 100);
color: rgb(255, 255, 255);
border-radius: 5px;
</string>
       </property>
       <property name="text">
        <string>Quiz</string>
       </property>
       <property name="icon">
        <iconset resource="imagenes.qrc">
         <normaloff>:/images/images/quiz.png</normaloff>:/images/images/quiz.png</iconset>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="botonVolver">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style: solid;
border-width: 1px;
;
background-color: rgb(0, 120, 100);
color: rgb(255, 255, 255);
border-radius: 5px;

</string>
       </property>
       <property name="text">
        <string>Menu</string>
       </property>
       <property name="icon">
        <iconset resource="imagenes.qrc">
         <normaloff>:/images/images/Menu.png</normaloff>:/images/images/Menu.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="BotonAnterior">
       <property name="styleSheet">
        <string notr="true">border-style: solid;
border-width: 1px;
;
background-color: rgb(129, 120, 255);
background-color: rgb(255, 176, 248);
color: rgb(255, 255, 255);
background-color: rgb(7, 99, 157);
border-radius: 5px;
</string>
       </property>
       <property name="text">
        <string>Anterior</string>
       </property>
       <property name="icon">
        <iconset resource="imagenes.qrc">
         <normaloff>:/images/images/Anterior.png</normaloff>:/images/images/Anterior.png</iconset>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="BotonSiguiente">
       <property name="styleSheet">
        <string notr="true">border-style: solid;
border-width: 1px;
;
background-color: rgb(129, 120, 255);
background-color: rgb(255, 176, 248);
color: rgb(255, 255, 255);
background-color: rgb(7, 99, 157);
border-radius: 5px;

</string>
       </property>
       <property name="text">
        <string>Siguiente</string>
       </property>
       <property name="icon">
        <iconset resource="imagenes.qrc">
         <normaloff>:/images/images/Siguiente.png</normaloff>:/images/images/Siguiente.png</iconset>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
